### DISCUSSION TITLE
Turn off Inlay Hints: Call Argument Names for hidden argument names

### DISCUSSION THREAD

**Discussion by jordan-jack-schneider** [microsoft/pylance-release#4527] [Enhancement]
Inlay hints for functions with hidden argument names are often obtrusive and make it harder to read the code. It would be nice to have an option to have call argument names in general, but not for hidden arguments (i.e. those starting with _ or __, and those that can never be provided as kwargs).
<img width="167" alt="Screenshot 2023-06-21 at 10 24 33" src="https://github.com/microsoft/pylance-release/assets/124743827/7d5629c4-6dc7-4751-8cc8-dd976fae69bf">

**Comment by debonte** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
